app/components/common/ListHeader.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { StyleSheet, TouchableOpacity, Image } from 'react-native'
+import colors from '../../config/colors'
```

Comment:
Little tip. The project has a package.json file included that will let you import using app/config/colors. 
Helps if you ever split move code into or out of folders.

Reply:
ah yeah, I've done that in a few of the newer files, good to know, can definitely use that on econform

Reply:
Yeah I just seen after I left that.